diff --git a/node_modules/react-router/dist/development/chunk-HA7DTUK3.mjs b/node_modules/react-router/dist/development/chunk-HA7DTUK3.mjs
index ac09151..70f7449 100644
--- a/node_modules/react-router/dist/development/chunk-HA7DTUK3.mjs
+++ b/node_modules/react-router/dist/development/chunk-HA7DTUK3.mjs
@@ -5328,13 +5328,7 @@ async function loadRouteModule(route, routeModulesCache) {
     return routeModulesCache[route.id];
   }
   try {
-    let routeModule = await import(
-      /* @vite-ignore */
-      /* webpackIgnore: true */
-      route.module
-    );
-    routeModulesCache[route.id] = routeModule;
-    return routeModule;
+    throw new Error("Route modules are not supported by older devices.");
   } catch (error) {
     console.error(
       `Error loading route module \`${route.module}\`, reloading page...`
@@ -6214,11 +6208,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       return fetchServerHandler(singleFetch);
     }
     function prefetchModule(modulePath) {
-      import(
-        /* @vite-ignore */
-        /* webpackIgnore: true */
-        modulePath
-      );
+      throw new Error("Route modules are not supported by older devices.");
     }
     function prefetchRouteModuleChunks(route2) {
       if (route2.clientActionModule) {
@@ -6326,11 +6316,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       } else if (route.clientLoaderModule) {
         dataRoute.loader = async (args, singleFetch) => {
           invariant2(route.clientLoaderModule);
-          let { clientLoader } = await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientLoaderModule
-          );
+          throw new Error("Route modules are not supported by older devices.");
           return clientLoader({
             ...args,
             async serverLoader() {
@@ -6351,11 +6337,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
         dataRoute.action = async (args, singleFetch) => {
           invariant2(route.clientActionModule);
           prefetchRouteModuleChunks(route);
-          let { clientAction } = await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientActionModule
-          );
+          throw new Error("Route modules are not supported by older devices.");
           return clientAction({
             ...args,
             async serverAction() {
diff --git a/node_modules/react-router/dist/production/chunk-Z4EF7MSU.mjs b/node_modules/react-router/dist/production/chunk-Z4EF7MSU.mjs
index 422c090..13ba707 100644
--- a/node_modules/react-router/dist/production/chunk-Z4EF7MSU.mjs
+++ b/node_modules/react-router/dist/production/chunk-Z4EF7MSU.mjs
@@ -5328,13 +5328,7 @@ async function loadRouteModule(route, routeModulesCache) {
     return routeModulesCache[route.id];
   }
   try {
-    let routeModule = await import(
-      /* @vite-ignore */
-      /* webpackIgnore: true */
-      route.module
-    );
-    routeModulesCache[route.id] = routeModule;
-    return routeModule;
+    throw new Error("Route modules are not supported by older devices.");
   } catch (error) {
     console.error(
       `Error loading route module \`${route.module}\`, reloading page...`
@@ -6214,11 +6208,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       return fetchServerHandler(singleFetch);
     }
     function prefetchModule(modulePath) {
-      import(
-        /* @vite-ignore */
-        /* webpackIgnore: true */
-        modulePath
-      );
+      throw new Error("Route modules are not supported by older devices.");
     }
     function prefetchRouteModuleChunks(route2) {
       if (route2.clientActionModule) {
@@ -6326,11 +6316,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
       } else if (route.clientLoaderModule) {
         dataRoute.loader = async (args, singleFetch) => {
           invariant2(route.clientLoaderModule);
-          let { clientLoader } = await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientLoaderModule
-          );
+          throw new Error("Route modules are not supported by older devices.");
           return clientLoader({
             ...args,
             async serverLoader() {
@@ -6351,11 +6337,7 @@ function createClientRoutes(manifest, routeModulesCache, initialState, ssr, isSp
         dataRoute.action = async (args, singleFetch) => {
           invariant2(route.clientActionModule);
           prefetchRouteModuleChunks(route);
-          let { clientAction } = await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientActionModule
-          );
+          throw new Error("Route modules are not supported by older devices.");
           return clientAction({
             ...args,
             async serverAction() {
